client/src/components/conclusions-list.tsx

```diff
@@ -57,11 +57,11 @@ const ConclusionCard = (props: {
         <TextField
           margin="normal"
           data-cy="edit-conclusion"
-          label={`Conclusion ${idx + 1}`}
+          label={`Message ${idx + 1}`}
           multiline
           maxRows={4}
           fullWidth
-          placeholder="Add a conclusion statement, e.g. 'In summary,  RGB colors are red, green, and blue'"
+          placeholder="Add a concluding statement/summary."
```

Comment:
why the change in text?

Reply:
We made an overhaul to a lot of the phrasing and to make it more aligned with what teacher's are used to seeing in their learning management systems. I understand how the example text could still be useful so if we wanted to push for keeping it in we can.